modules/acquia_cms_headless/acquia_cms_headless.info.yml

```diff
@@ -9,12 +9,9 @@ dependencies:
   - jsonapi_extras:jsonapi_extras
   - jsonapi_menu_items:jsonapi_menu_items
   - jsonapi_resources:jsonapi_resources
```

Comment:
@apathak18 you can also remove jsonapi_resources:jsonapi_resources as jsonapi_menu_items have this dependency.

cc @vishalkhode1